Cargo.toml

```diff
@@ -82,44 +79,5 @@ clippy.redundant_feature_names = "warn"
 clippy.wildcard_dependencies = "warn"
 # END LINEBENDER LINT SET
 
-[features]
-default = ["std"]
-std = []
-libm = ["dep:libm"]
-mint = ["dep:mint"]
-serde = ["smallvec/serde", "dep:serde"]
-schemars = ["schemars/smallvec", "dep:schemars"]
-
-[dependencies]
-smallvec = "1.13.2"
-
-[dependencies.arrayvec]
-version = "0.7.6"
-default-features = false
-
-[dependencies.libm]
-version = "0.2.11"
-optional = true
-
-[dependencies.mint]
-version = "0.5.9"
-optional = true
-
-[dependencies.schemars]
-version = "0.8.21"
-optional = true
-
-[dependencies.serde]
-version = "1.0.215"
-optional = true
-default-features = false
-features = ["alloc", "derive"]
-
-# This is used for research but not really needed; maybe refactor.
-[dev-dependencies]
-rand = "0.9.0"
-
-[target.wasm32-unknown-unknown.dev-dependencies]
-# We have a transitive dependency on getrandom and it does not automatically
-# support wasm32-unknown-unknown. We need to enable the wasm_js feature.
-getrandom = { version = "0.3.1", features = ["wasm_js"] }
+[workspace.dependencies]
```

Comment:
I know our "standard" is to put `workspace.dependencies` after `workspace.lints`, but I wonder if doing the other way around would be clearer; I don't recall having any discussion about this.

Not a blocking concern

Reply:
Agreed, moving it in this PR.